### PR TITLE
Add tests for ibft

### PR DESCRIPTION
**Pull request by rvykydal**
Requires ipxe.lkrn image (ipxe-bootimgs package)
Requires isoinfo tool (genisomage package) - should be pulled in by lorax

### CODE REVIEW

File: iscsi.ks.in [hunk 1]
```diff
@@ -21,7 +21,8 @@ keyboard us
 lang en
 timezone America/New_York
 rootpw qweqwe
-shutdown
+#DBG
+#shutdown
```
Comment (vpodzime) [inline iscsi.ks.in]: Is this change intentional?

Comment (rvykydal) [inline iscsi.ks.in]: No, thank you for the catch.
